modules/local/orthofinder_phylohogs.nf

```diff
@@ -0,0 +1,73 @@
+process ORTHOFINDER_PHYLOHOGS {
+    // label 'process_highthread'
+    label 'process_medium'
+    container "${ workflow.containerEngine == 'docker' ? 'austinhpatton123/orthofinder-2.5.4_r-4.2.2' : 
+        '' }"
+
+    publishDir(
+        path: "${params.outdir}/orthofinder_phylohogs/",
+        mode: 'copy',
+        saveAs: { fn -> fn.substring(fn.lastIndexOf('/')+1) },
+    )
+    
+    input:
+    path species_tree    // Rooted species tree inferred using SpeciesRax
+    path speciesrax_gfts // Reconciled gene family trees used in species tree inference
+    path generax_gfts    // Remaining reconciled gene family trees
+    
+
+    output:
+    path "*" , emit: phylohogs
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
```

Comment:
You include this with bunch of them, but majority don't use it. Are these needed? I didn't want to remove without knowing but can you confirm if this is needed or not?

Reply:
I'm not confident, but I suspect this has something to do with an option where you can specify additional arguments/parameters in the modules.json file. So, this is like one of three ways to specify user-input parameters that are not hard-coded as parameter options with default values?

Reply:
Regardless, I think it's only necessary/useful for some - this one not being one, so I'll get rid of here.